app/assets/javascripts/training/components/Alert.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+
+export default function Alert() {
```

Comment:
This file and component needs a better name. It's not a general alert (which I would expect to take props to control what message it shows), it's a very specific one with a message built in.